sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md

```diff
@@ -24,6 +24,7 @@ Invoke-SPOSiteSwap
     -TargetUrl <string>
     -ArchiveUrl <string>
     [-DisableRedirection]
+    [-Force]
```

Comment:
I think there is also a -NoWait parameter that should be added to the article as it's not a common parameter.

Reply:
We don't do anything specific with the -NoWait so this has not been called out specifically.

Reply:
Ok, we usually add these parameters and put something like "This parameter is reserved for internal Microsoft use" to avoid future issues/questions. Are you agree? I can add it to the same PR.

Reply:
Personally I'm too concerned, but happy to go with your guidance here. So if it will help reduce issues/questions then "yes" :)

Reply:
Thank you!